### PR TITLE
MAT-4510: Remove Stratification Button

### PR DESCRIPTION
**Pull request by ethankaplan**
## MADiE PR

Jira Ticket: [MAT-4511](https://jira.cms.gov/browse/MAT-4511)
(Optional) Related Tickets: [MAT-4510](https://jira.cms.gov/browse/MAT-4510)

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [x] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [x] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

**Comment by gregory-akins** [comment]
Try to get that coverage back above 90%

**Review by gregory-akins — DISMISSED** [review]

**Comment by gregory-akins** [comment]
@ethankaplan will you see if you can get the coverage back over 90%?

**Comment by ethankaplan** [comment]
Some tests I'm not a part of are being skipped, most of the code I've added is covered

File: src/components/measureGroups/MeasureGroups.tsx [hunk 1]
```diff
@@ -702,127 +720,170 @@ const MeasureGroups = () => {
...
+                                <Row>
+                                  <Col>
+                                    <FieldLabel htmlFor="stratification-select">
+                                      Stratification {i + 1} {}
+                                      {formik.values.stratifications.length >
+                                        2 &&
+                                        visibleStrats > 2 && (
+                                          <span>
+                                            &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+                                            <Button
```
Comment (RohitKandimalla) [inline src/components/measureGroups/MeasureGroups.tsx:750]: Can we not use buttons from madie-components, and use madie-design-systems instead ?

File: src/components/measureGroups/MeasureGroups.tsx [hunk 2]
```diff
@@ -702,127 +720,170 @@ const MeasureGroups = () => {
...
+                                              buttonTitle="Remove"
+                                              variant="white"
+                                              buttonSize="xs"
+                                              data-testid="remove-strat-button"
+                                            />
+                                          </span>
+                                        )}
+                                    </FieldLabel>
+
+                                    <TextField
```
Comment (RohitKandimalla) [inline src/components/measureGroups/MeasureGroups.tsx:771]: Also for the select, we have a reusable component available in madie-design-system which should match the figma designs.

**Review by RohitKandimalla — CHANGES_REQUESTED** [review]

File: src/components/measureGroups/MeasureGroups.tsx (hunk 1 above)
Comment (ethankaplan) [reply src/components/measureGroups/MeasureGroups.tsx:734]: to come later

File: src/components/measureGroups/MeasureGroups.tsx (hunk 2 above)
Comment (ethankaplan) [reply src/components/measureGroups/MeasureGroups.tsx:754]: to come later

File: src/components/measureGroups/MeasureGroups.tsx (hunk 1 above)
Comment (ethankaplan) [reply src/components/measureGroups/MeasureGroups.tsx:750]: Done!

File: src/components/measureGroups/MeasureGroups.tsx (hunk 2 above)
Comment (ethankaplan) [reply src/components/measureGroups/MeasureGroups.tsx:755]: Struggling to get select to work, but working on it. Can be merged as is right now and select be changed later

File: src/components/measureGroups/MeasureGroups.tsx [hunk 3]
```diff
@@ -29,6 +29,8 @@ import {
 } from "@madie/madie-util";
 import MultipleSelectDropDown from "./MultipleSelectDropDown";
 import MeasureGroupsWarningDialog from "./MeasureGroupWarningDialog";
+
+import { values } from "lodash";
```
Comment (RohitKandimalla) [inline src/components/measureGroups/MeasureGroups.tsx]: Looks like Lodash is already imported and also it looks like this is not used anywhere.

File: src/components/measureGroups/MeasureGroups.tsx [hunk 4]
```diff
@@ -121,6 +123,12 @@ export const EmptyStrat = {
   association: "",
   id: "",
 };
+export const DeleteStrat = {
+  cqlDefinition: "delete",
+  description: "delete",
+  association: "delete",
+  id: "",
+};
```
Comment (RohitKandimalla) [inline src/components/measureGroups/MeasureGroups.tsx:129]: A variable naming convention is usually a camel case, Correct me if I am wrong about it in TypeScript.

**Review by RohitKandimalla — DISMISSED** [review]

**Review by RohitKandimalla — APPROVED** [review]

**Pull request merged**